nemo/export/trt_llm/nemo/nemo.py

```diff
@@ -0,0 +1,283 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Referrence impl in tensorrt_llm: examples/gpt/utils/nemo.py."""
+import functools
+import logging
+import os
+import pathlib
+import tarfile
+import typing
+
+import torch
+import yaml
+from transformers import FalconConfig, GPT2Config, LlamaConfig
+
+from .convert import cpu_map_location, gpu_map_location
+
+LOGGER = logging.getLogger("NeMo")
+
+
+def nemo_to_llm_config(nemo_model_config, vocab_size, eos_id, bos_id, decoder_type):
```

Comment:
Please rename according to previous comments

Reply:
These are actually internal functions to TRT-LLM export, so really not that important since it won't be reused anywhere else. But this has llm in it, no?